tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -148,11 +148,12 @@ public function clearResponseXmlObject():void {
 	}
 
 	/**
+	 * @param SimpleXMLElement|null $xmlObject
 	 *
 	 * @return string the etag or an empty string if the getetag property does not exist
 	 */
-	public function getEtagFromResponseXmlObject():string {
-		$xmlObject = $this->getResponseXmlObject();
+	public function getEtagFromResponseXmlObject(?SimpleXMLElement $xmlObject = null):string {
+		$xmlObject = $xmlObject ?? $this->getResponseXmlObject();
```

Comment:
I think you should use this method instead (which should be from FeatureContext)
```suggestion
		$xmlObject = $xmlObject ?? $this->getResponseXml();
```

Reply:
not passing on using the method getResponseXml() 
https://drone.owncloud.com/owncloud/ocis/27127/58/5
but it passes using the method getResponseXmlObject()

Reply:
`$this->featureContext->getResponseXml()` should work. We need to get rid of similar methods